### ISSUE TITLE
Client side

### ISSUE THREAD

**Issue by Shuunen** [avxto/pigeon-js#2]
Hi,

This lib is meant to be used server side but is it possible to have a clent-side port ? 

Thanks

**Comment by avxto** [comment]
Yes, it is possible. The lib is not using any core modules, so it can be done using browserify.

**Comment by Shuunen** [comment]
Cool thanks :)

**Comment by avxto** [comment]
No problem!
